datameta/api/apikeys.py

```diff
@@ -103,3 +130,48 @@ def post(request:Request) -> UserSession:
             raise HTTPUnauthorized()
 
     return generate_api_key(request, auth_user, label)
+
+
+@view_config(
+    route_name="user_id_keys", 
+    renderer='json', 
+    request_method="GET", 
+    openapi=True
+)
+def get_user_keys(request:Request) -> UserSession:
+    """Request new ApiKey"""
+    
+    auth_user = security.revalidate_user(request)
+    if not auth_user:
+        raise HTTPUnauthorized()
```

Comment:
```suggestion
```
`security.revalidate_user()` already throws `HTTPUnauthorized`, so this code will never be reached and can be omitted